src/qt/masternodelist.h

```diff
@@ -85,5 +89,10 @@ private Q_SLOTS:
     void on_startMissingButton_clicked();
     void on_tableWidgetMyMasternodes_itemSelectionChanged();
     void on_UpdateButton_clicked();
+
+    CDeterministicMNCPtr GetSelectedDIP3MN();
```

Comment:
This is not a slot, it's just a regular private function, should be in previous section (e.g. line 80).